CMSgov/mint-app
# [EASI-2373] - Solution Details

# EASI-2373

## Changes and Description

- Solution details view and component
- Subtask component and links (links don't work yet)
- Link documents table (links don't work yet)
- Update existing solutions 
    - Alerts for updating and saving solutions
- Storybooks for `Subtasks` and `SolutionDetailCard`
- Cypress/unit testing

Some of the work for updating status details is covered in another ticket.  The scope of this ticket is covering the solution detail page and updating existing solutions

![Screenshot 2023-01-03 at 9 47 37 AM](https://user-images.githubusercontent.com/95709965/210380429-26c0b5b3-5580-4646-b2e4-466fe18e0840.png)

![Screenshot 2022-12-30 at 9 44 00 AM](https://user-images.githubusercontent.com/95709965/210082458-a93443fa-e956-4501-965c-659b52908cda.png)

![Screenshot 2022-12-30 at 9 44 12 AM](https://user-images.githubusercontent.com/95709965/210082484-465a22c3-5026-42c2-83b8-35f9efd5bfff.png)


## How to test this change

- Trigger an operational need selection by answering a task list question
- Select a solution/s and save details (can also be a custom solution)
- From the IT solution tracker table, select "View details"
- Verify correct info
- Select "Update solutions for this operational need" and add or remove solutions

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
